requirements.txt

```diff
@@ -0,0 +1 @@
+"core" {>= "0.14.1"}
```

Comment:
Kaj je to zaen paket?

Reply:
Sorry, je ostalo od enih izmed prejšnih implementacij grafa, hvala da ste opazili. Odstranjujem...

Reply:
b9c32cd0d7c41ebb0fba09e79cfbd1670f881bb3